golang/channel.go

```diff
@@ -209,6 +209,14 @@ func (ch *Channel) GetSubChannel(serviceName string) *SubChannel {
 	return ch.registerNewSubChannel(serviceName)
 }
 
+// Peers returns the PeerList for the channel.
+func (ch *Channel) Peers() *PeerList {
+	ch.mutable.mut.RLock()
```

Comment:
Do you need the read mut?  I don't see where we ever update this  - looks like it is set in the ctor and never modified after.